pkgs/by-name/sc/scopehal-apps/package.nix

```
@@ -0,0 +1,100 @@
+{
+  stdenv,
+  breakpointHook,
+  fetchFromGitHub,
+  cmake,
+  git,
+  lib,
+  pkg-config,
+  libsigcxx,
+  gtkmm3,
+  cairomm,
+  yaml-cpp,
+  catch2,
+  glfw,
+  libtirpc,
+  liblxi,
+  glew,
+  libllvm,
+  libdrm,
+  elfutils,
+  libxcb,
+  zstd,
+  libxshmfence,
+  wrapGAppsNoGuiHook,
+  xcbutilkeysyms,
+  systemd,
+  shaderc,
+  vulkan-headers,
+  vulkan-loader,
+  vulkan-tools,
+  glslang,
+  spirv-tools,
+  ffts,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "scopehal-apps";
+  version = "unstable-2024-08-22";
+
+  src = fetchFromGitHub {
+    owner = "ngscopeclient";
+    repo = "scopehal-apps";
+    rev = version;
```

```suggestion
    rev = "e75c0bd1eed4c057bfbbfa6a75a211a88d30c934";
```
else it won't build (`fatal: couldn't find remote ref refs/tags/unstable-2024-08-22`)